README.md

```diff
@@ -249,6 +249,45 @@ to the user.
 If no middleware manages the error, the lambda execution fails reporting the unmanaged error.
 
 
+### Async/Await support
+
+Middy allows you to write *async/await handlers*. In *async/await handlers* you don't
+have to invoke the callback but just return the output (in case of success) or
+throw an error (in case of failure).
+
+We believe that this feature makes handling asynchronous logic easier to reason about
+and asynchronous code easier to read.
+
+Take the following code as an example:
+
+```javascript
+middy(async (event, context) => {
+  await someAsyncStuff()
+  await someOtherAsyncStuff()
+
+  return ({foo: bar})
+})
+```
+
+this code is equivalent to:
+
+```javascript
+middy(async (event, context, callback) => {
+  someAsyncStuff()
+    .then(() => {
+      return someOtherAsyncStuff()
+    })
+    .then(() => {
+      callback(null, {foo: bar})
+    })
+})
+```
+
+We believe that the async/await version is easier to read and reason about.
```

Comment:
This sentence is redundant based on sentence above I think.

Reply:
Good point!